src/common.c

```diff
@@ -2648,6 +2648,8 @@ bool calculate_aggregate_bounds(tree_t expr, range_kind_t *kind,
 
    type_t type = tree_type(expr);
    type_t index_type = index_type_of(type, 0);
+   if (index_type == NULL || type_is_none(index_type))
+      return false;
```

Comment:
```suggestion
   if (index_type == NULL || type_is_none(index_type))
      return false;

```

Looks a bit neater with a blank line.